readme.md

```diff
@@ -1 +1,96 @@
 # Ticket booking app
+
+Ticket booking system build with akka-http.
+
+1. [About](#about)
+1. [Installing](#installing)
+1. [API](#api)
+1. [Dependencies](#dependencies)
+
+## About
+This system was build with akka-http high level API. I made following assumptions in this project:
+* each row in screening room has same number of seats
+* no authentication
+
+Besides akka-http I used following libraries, technologies:
+* *slick* for database connection,
+* *circe* for json parsing,
+* *posgresql* as database server,
+* *scalatest* for unit testing.
+
+
+## Installing
+
+### Requirements
+* Scala 2.13
+* Java 1.8 or higher
+* [Optional] PosgreSQL server
+
+### Steps
+
+1. **Clone this repository**
+```git clone https://github.com/rochala/ticket-booking.git```
+1. **Change directory to ticket-booking**
+```cd ticket-booking```
+1. **Checkout to development branch**
+```git checkout development```
+1. **Database configuration**
+You can either use external database hosted on my server ( configuration is provided in email ) and skip next sections or
+host it locally on your machine.
```

Comment:
It would be nice to have postgres in docker or db in memory for tests/development puprposes